spec/spec_helper.rb

```diff
@@ -2,6 +2,6 @@
 require 'chefspec/berkshelf'
 
 RSpec.configure do |config|
-  config.color_enabled = true
+  config.color     = true
```

Comment:
As far as I can tell. When I run it with `color_enabled` it raises an error:
```shell
 pantry/spec/spec_helper.rb:5:in `block in <top (required)>': undefined method `color_enabled=' for #<RSpec::Core::Configuration:0x007ff5038299c0> (NoMethodError)
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core.rb:81:in `configure'
	from /Users/pwelch/Code/github/opensource/pantry/spec/spec_helper.rb:4:in `<top (required)>'
	from /Users/pwelch/Code/github/opensource/pantry/spec/recipes/default_spec.rb:1:in `require'
	from /Users/pwelch/Code/github/opensource/pantry/spec/recipes/default_spec.rb:1:in `<top (required)>'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `load'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `block in load_spec_files'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `each'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `load_spec_files'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:96:in `setup'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:84:in `run'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:69:in `run'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:37:in `invoke'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/exe/rspec:4:in `<top (required)>'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/bin/rspec:23:in `load'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/bin/rspec:23:in `<main>'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `eval'
	from /Users/pwelch/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `<main>'
```
Setting to color seems to make Rspec 3 happy.